plugins/lang-groovy/src/test/java/org/elasticsearch/script/groovy/GroovySecurityTests.java

```diff
@@ -107,6 +107,9 @@ public void testEvilGroovyScripts() throws Exception {
         // AccessControlException[access denied ("java.io.FilePermission" "<<ALL FILES>>" "execute")]
         assertFailure("def methodName = 'ex'; Runtime.\"${'get' + 'Runtime'}\"().\"${methodName}ec\"(\"touch /tmp/gotcha2\")");
 
+        // AccessControlException[access denied ("java.lang.RuntimePermission" "modifyThreadGroup")]
+        assertFailure("t = new Thread({ println 3 }); t.start(); t.join()");
```

Comment:
can you add a comment the we are not allowed to start a thread anymore and maybe remove the t.join() since it's confusing to me why it needs this